### PR TITLE
Fix in meeting dialog responsiveness

### PR DESCRIPTION
**Pull request by heath-hamilton**
Minor edits to align with design UI kit and link to implementation details page.

### CODE REVIEW

File: msteams-platform/apps-in-teams-meetings/design/designing-apps-in-meetings.md [hunk 1]
```diff
@@ -146,19 +146,18 @@ There are two header variants. When possible, use the variant with the avatar to
 
 In-meeting dialogs can vary in size to account for different scenarios. Make sure to maintain padding and component sizes.
 
-* **Width**: The iframe width of the dialog is an absolute value within the range you specify.
-* **Height**: The iframe height of the dialog is an absolute value within the range you specify.
+* **Width**: You can specify the width of the dialog's iframe anywhere within the supported size range.
+* **Height**: You can specify the height of the dialog's iframe anywhere within the supported size range. Users can scroll vertically if your app content exceeds the maximum height.
```
Comment (surbhigupta12) [inline msteams-platform/apps-in-teams-meetings/design/designing-apps-in-meetings.md:150]: @Trinetra-MSFT, please confirm on the technical accuracy for the following sentence:

Users can scroll vertically if your app content exceeds the maximum height.

Comment (surbhigupta12) [inline msteams-platform/apps-in-teams-meetings/design/designing-apps-in-meetings.md:150]: @Trinetra-MSFT, please confirm on the technical accuracy for the following sentence:

Users can scroll vertically if your app content exceeds the maximum height.

**Review by KirtiPereira — CHANGES_REQUESTED** [review]

File: msteams-platform/apps-in-teams-meetings/design/designing-apps-in-meetings.md (hunk 1 above)
Comment (heath-hamilton) [inline msteams-platform/apps-in-teams-meetings/design/designing-apps-in-meetings.md:150]:
```suggestion
* **Height**: You can specify the height of the dialog's iframe anywhere within the supported size range. You also can allow users to scroll vertically if your app content exceeds the maximum height.
```
@KirtiPereira I changed the second sentence based on a discussion with @aosolis. Vertical scrolling is supported, but it's up to the dev to set up.

[the remark this answers is not in the document]
Comment (heath-hamilton) [reply msteams-platform/apps-in-teams-meetings/design/designing-apps-in-meetings.md:150]: @KirtiPereira 
```suggestion
* **Height**: You can specify the height of the dialog's iframe anywhere within the supported size range. You also can allow users to scroll vertically if your app content exceeds the maximum height.
```
@KirtiPereira I changed the second sentence based on a discussion with @aosolis. Vertical scrolling is actually supported, but it's up to the dev to set up.

File: msteams-platform/bots/how-to/update-and-delete-bot-messages.md [hunk 2]
```diff
@@ -142,17 +155,16 @@ for each activity_id in _list:
...
+| N/A | An HTTP status code that indicates the outcome of the operation. Nothing is specified in the body of the response. |
 
 ---
 
 ## Code samples
 
 The official conversation basics are as follows:
 
-| Sample Name           | Description                                                                      | .NET    | JavaScript   | Python  |
+| Sample Name           | Description                                                                      | C#    | JavaScript   | Python  |
```
[the remark this answers is not in the document]
Comment (heath-hamilton) [reply msteams-platform/bots/how-to/update-and-delete-bot-messages.md:168]:
```suggestion
| Sample name           | Description                                                                      | .NET    | JavaScript   | Python  |
```
Done. Also corrected title case in "Sample Name" to align with our style.

**Review by heath-hamilton — COMMENTED** [review]
@KirtiPereira I believe I addressed all your comments. Thanks in advance!

**Comment by heath-hamilton** [comment]
@surbhigupta12 This PR is ready to merge. Thanks in advance!

**Review by KirtiPereira — APPROVED** [review]
Approved

**Pull request merged**